content/maxwell3_fdem/emi_tutorial/index.rst

```diff
@@ -0,0 +1,44 @@
+.. _emi_tuotorial_index:
+
+EM induction tutorial
+=====================
+
+The fundamental principles of electromagnetic (EM) induction can be understood with a simple geometry involving three elements each represented by a loop of conducting wire as shown in :numref:`Concepts_3loops`.
+
+- Loop1: is the transmitter. It carries a current that produces an associated magnetic field. This magnetic field is called the primary field. For induction experiments the current in the transmitter varies with time and hence the magnetic field varies with time.
+
+- Loop2: represents a conductive body. The  time varying magnetic field generates an  Electromotive Force (EMF) and hence a current in the conductor. These are called secondary currents. The generate secondary magnetic fields.
+
+- Loop3: is the receiver: This measures magnetic field which consists of the sum of the primary and secondary fields.
+
+.. figure:: ./images/Concepts_3loops.png
+   :align: center
+   :scale: 60%
+   :name: Concepts_3loops
+
+   Conceptual diagrams for EM inductions. Top panel shows excitation of the conductor using induction, and botton panel shows corresponding 3-loops system.
+
+.. note::
+	Geophysical goal: remove the primary field from the measurements and interpret the data to find information about the buried conductor.
```

Comment:
here, I think it is worthwhile to clarify "information" ie: physical properties and location